datacleaner/datacleaner.py

```diff
@@ -72,11 +83,18 @@ def autoclean(input_dataframe, drop_nans=False, copy=False):
 
         # Encode all strings with numerical equivalents
         if str(input_dataframe[column].values.dtype) == 'object':
-            input_dataframe[column] = LabelEncoder().fit_transform(input_dataframe[column].values)
+            if encoder is None:
+                input_dataframe[column] = encoder().fit_transform(input_dataframe[column].values)
```

Comment:
I suppose it would break support for sklearn.preprocessing.PolynomialFeatures as well, but I think that's okay. The goal of datacleaner isn't necessarily to transform the features significantly, as HashingEncoder and PolynomialFeatures does.